packages/kbn-securitysolution-ecs/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "target/types",
+    "types": [
+      "jest",
+      "node",
+      "react"
```

Comment:
Curious if the `react` compiler is necessary.

Reply:
Good point, probably not jest either